robottelo/host_helpers/cli_factory.py

```diff
@@ -0,0 +1,285 @@
+import datetime
+import inspect
+import pprint
+import random
+from functools import lru_cache
+from functools import partial
+
+from fauxfactory import gen_alpha
+from fauxfactory import gen_alphanumeric
+from fauxfactory import gen_choice
+from fauxfactory import gen_ipaddr
+from fauxfactory import gen_mac
+from fauxfactory import gen_netmask
+
+from robottelo import constants
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.config import settings
+from robottelo.helpers import update_dictionary
+
+# import os
```

Comment:
placeholder comment to cleanup

Reply:
placeholder, i had to comment out some imports that aren't used since i removed a large chunk of methods i'm still working on. the vast majority of these imports will be used in the next update